src/ts/state/selectors.ts

```diff
@@ -123,11 +110,19 @@ export const selectAllAutoFixableMatches = <TMatch extends IMatch>(
     _ => _.replacement && _.replacement.text === _.message
   );
 
-export const selectHasError = <TMatch extends IMatch>(
+export const selectHasGeneralError = <TMatch extends IMatch>(
   state: IPluginState<TMatch>
-): boolean =>
-  !!state.requestErrors && state.requestErrors.length > 0;
+): boolean => {
+  const generalErrors = state.requestErrors.filter(_ => _.type === "GENERAL_ERROR");
+  return !!state.requestErrors && generalErrors.length > 0};
```

Comment:
Is the `!!state.requestErrors` check necessary? Looking at the `IState` type, it looks like this will always be true, as it's a non-optional array? (Same for `selectHasAuthError` below)